java/src/jni/h5sImp.c

```diff
@@ -1340,6 +1340,389 @@ Java_hdf_hdf5lib_H5_H5Sget_1regular_1hyperslab(JNIEnv *env, jclass clss, jlong s
     return;
 } /* end Java_hdf_hdf5lib_H5_H5Sget_1regular_1hyperslab */
 
+/*
+ * Class:     hdf_hdf5lib_H5
+ * Method:    H5Sselect_copy
+ * Signature: (JJ)V
+ */
+JNIEXPORT void JNICALL
+Java_hdf_hdf5lib_H5_H5Sselect_1copy(JNIEnv *env, jclass clss, jlong dst_id, jlong src_id)
+{
+    hssize_t retVal = -1;
+
+    UNUSED(clss);
+
+    if ((retVal = H5Sselect_copy((hid_t)dst_id, (hid_t)src_id)) < 0)
```

Comment:
H5Sselect_copy returns an herr_t, not an hssize_t. Another case where the API should return the error and not void.

Reply:
Fixed.